storage/views.py

```diff
@@ -19,7 +19,7 @@ def check_is_client_exists(serializer, pk=None):
     client_n = serializer.validated_data['client_name']
     client_d_cr = serializer.validated_data['createdAt']
 
-    client = Client.objects.filter(client_name=client_n).first()
+    client = Client.objects.filter(pk=pk).first()
```

Comment:
This will be more correct:
```
    client = Client.objects.filter(client_name=client_n,
                                   createdAt=client_d_cr,
                                   ).first()
```